manage.py

```diff
@@ -586,3 +586,153 @@ def merge_users(src_id, tgt_id, actor):
         user_id=acting_user.id,
         subject_id=tgt_id)
     print(message)
+
+
+@click.option('--org_id', help="Organization (site) ID")
+@click.option(
+    '--retired',
+    help="datetime for site's current protocol expiration,"
+         " format example: 2019-04-09 15:14:43")
```

Comment:
```suggestion
@click.option('--org_id', help="Organization (site) ID", required=True)
@click.option(
    '--retired',
    required=True,
    help="datetime for site's current protocol expiration,"
         " format example: 2019-04-09 15:14:43")
```

I think both these are required, so `required=True` should be passed [as a keyword arg](https://click.palletsprojects.com/en/7.x/options/#basic-value-options)

>By default, options are not required, however to make an option required, simply pass in required=True as an argument to the decorator.

Reply:
Regular (positional) [arguments](https://click.palletsprojects.com/en/7.x/arguments/) might make sense too, if there's some obvious ordering

Reply:
Yes, positional would make more POSIX sense, given they're required, but the ordering isn't obvious.  I like flagging them as required.